src/components/seo.tsx

```diff
@@ -58,35 +58,37 @@ export const SEO = ({
   };
 
   return (
-    <ThemeProvider theme={theme}>
-      <title>{seo.title}</title>
-      <meta charSet="utf-8" />
-      <meta name="description" content={seo.description} />
-      <meta name="image" content={seo.image} />
-      <meta name="keywords" content={seo.metaKeywords} />
+    <html lang="en">
+      <ThemeProvider theme={theme}>
+        <title>{seo.title}</title>
+        <meta charSet="utf-8" />
+        <meta name="description" content={seo.description} />
+        <meta name="image" content={seo.image} />
+        <meta name="keywords" content={seo.metaKeywords} />
 
-      <meta name="theme-color" content={theme.colors.background} />
-      <meta itemProp="name" content={seo.title} />
-      <meta itemProp="description" content={seo.description} />
+        <meta name="theme-color" content={theme.colors.background} />
```

Comment:
❓ Is this change related to the POC? (just for my personal knowledge)

Reply:
It's a fix due to the accessibility report.
If it'll work, I'll report it in another PR 😉